src/main/scala/org/broadinstitute/dsde/workbench/leonardo/model/ServiceAccountProvider.scala

```diff
@@ -0,0 +1,53 @@
+package org.broadinstitute.dsde.workbench.leonardo.model
+
+import com.typesafe.config.Config
+import org.broadinstitute.dsde.workbench.model.google.GoogleProject
+import org.broadinstitute.dsde.workbench.model.{UserInfo, WorkbenchEmail}
+
+import scala.concurrent.Future
+
+/**
+  * Provides service accounts needed by Leo.
+  *
+  * @param config any necessary configuration information.
+  */
+abstract class ServiceAccountProvider(config: Config) {
+  /**
+    * The service account email used for making Google Dataproc calls.
+    * Note this does NOT return a Future; we expect a constant value for this.
+    *
+    * @return service account email
+    */
+  def getLeoServiceAccount: WorkbenchEmail
```

Comment:
Is it OK for this to not return a Future? I'm expecting this to be a constant value, and not an asynchronous operation. It's easier to work with in Leo code if it's not a future.

Reply:
Ah that's a good point. Leo config currently has a path to the serviceAccountPemFile. Should we remove this interface method and expect people running Leo to just configure the Leo SA+key via config?